### ISSUE TITLE
Filtered & Wireless Block Extender can no longer insert into inventories

### ISSUE THREAD

**Issue by toticorrea** [Dynious/RefinedRelocation#364]
Hi! Yesterday I installed the new version and so far I had a couple of issues: all sorting chests do not recognize their previous filters. I had to re filter every one.

Wireless block extender do not let items go in and out to a Oven. I had this set up from previus versions working fine but now I cant fgure it out why is not taking the items out and neither let the coal get in.

Regards

**Comment by blay09** [comment]
That's odd, it's set up to convert all older filters to the new system and worked in my testing. Were they just completely gone?

**Comment by toticorrea** [comment]
Some yes, some not. And since I was working on something else yesterday, I went back to the older version so I could continuing working and postpone this problem for other day. And when I put the older version, all filters were gone. So I had to go back to an older saved world to get back everything in order, with the 1.0.8b version of Refined. Now that I am aware this can happen, I will finish what I am doing in the next couple of days and then I will install again the 1.1.4 version and take note exactly what all the issues are, so I can be more specific with you. Sounds good to you?
